Documentation/network-policies.md

```diff
@@ -0,0 +1,30 @@
+
+# Network policies
+
+[Network policies](https://kubernetes.io/docs/user-guide/networkpolicies/) allow you to restrict the ingress traffic between pods. 
+To keep your cluster safer, it's strongly recomended to enable network policies into prometheus namespace.
+
+# Example
+
+In this example we are closing all the inbound communication on the namespace monitoring and just allowing the necessary traffic.
+
+
+Follow the steps [here](http://docs.projectcalico.org/v1.5/getting-started/kubernetes/installation/) to install calico, also dont' forget to [enable network policy](https:/kubernetes.io/docs/user-guide/networkpolicies) in you k8s cluster.
```

Comment:
Are there other providers than calico for network policies? If so this would be the point to state that the solutions would need to be evaluated against each other and say that these examples were tested with calico.